src/Coaches/Edit/Update.elm

```diff
@@ -6,22 +6,66 @@ import Effects exposing (Effects)
 
 import Coaches.Edit.Models exposing (..)
 import Coaches.Edit.Actions exposing (..)
+import Coaches.Edit.Effects exposing (..)
 
 
 update : Action -> ViewModel -> (ViewModel, Effects Action)
-update action model =
-    case action of
-        NoOp -> (model, Effects.none)
+update action model = 
+  let 
+    coach = Debug.log "coach" model.coach
 
-        SaveCoach ->
-          let
-            updateMessage message =
-              { message | text = "Error when saving coach.", messageType = Error }
+    updateCoach newCoach = 
+      ({model | coach = newCoach}, Effects.none) 
+  in
+    case action of 
+      NoOp -> (model, Effects.none)
 
+      Save -> (model, savePlayers coach)
```

Comment:
If you change the name of `savePlayers` in Effects file, you have to change here too.